src/main/java/wromaciej/moviesrental/movies/service/CartService.java

```diff
@@ -22,7 +24,15 @@ public double calculateRentalRate(Collection<Integer> movieIds) {
             Movie movie = movieRepository.findOne(movieId);
             sum += movie.getRentalRate();
         }
-
         return sum;
     }
+
+    public double calculateRentalRateForDays(int movieId, int rentalDays){
+        if (rentalDays>=0){
+            return movieService.findMovie(movieId).getRentalRate()*rentalDays;
```

Comment:
That is really simple one liner, but generally it is a good practice to save mid-term variables with meaningful name: 
```
Movie movie = movieService.findMovie(movieId);
return movie.getRentalRate() * rentalDays;
```